main.c

```diff
@@ -90,14 +90,10 @@ static int compile(int compile_options, int argc, const char *argv[])
 	  script_print_vals(&tmp_stack);
   }
 
-  stack_serialize(&tmp_stack, buf, bufsize, &compiled_len);
-
   if (compile_options & COMPILE_SHOW_STACK_HEX) {
 	  if (compile_options & COMPILE_SHOW_LABELS)
 		printf("stack_hex   ");
-	  for(i = 0; i < compiled_len; ++i)
-		  printf("%02x", buf[i]);
-	  printf("\n");
+		stack_print_hex(&tmp_stack);
```

Comment:
I don't think this is correct? this should show the stack as a hexstring

Reply:
I'll roll it back.